README.md

```diff
@@ -1 +1,24 @@
 # Operator Marketplace
+
+## Deploying the Marketplace Operator
+### Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Be logged into an OpenShift cluster as a user with the RBAC permissions required to run this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+2. Have created the CatalogSourceConfig custom resource defined within the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) project.
```

Comment:
`OLM` is a prerequisite, I think we want to ask the user to install `OLM`. Just registering the `CatalogSource` CRD will make marketplace component work, but you won't have the end to end experience.  

If you want to stick to just registering the CRD then:
 - There is a typo - it should be `CatalogSource`
- Also please include the file name in OLM repository. 
- Also clarify that you won't have end to end experience.

Reply:
Saying OLM is a prerequisite and pointing at the OLM install doc is the better option

Reply:
Updated doc to state that OLM is a prerequisite.